obal/data/scratch_build.yml

```diff
@@ -3,17 +3,24 @@
   serial: 1
   any_errors_fatal: false # don't bomb out the entire playbook if one host (i.e. package) fails
   gather_facts: no
-  roles:
-    - git_annex_setup
-    - setup_sources
-    - ensure_package
-  tasks:
+  pre_tasks:
     - name: 'Set tito_releasers'
       set_fact:
-        tito_releasers: "{{ releasers | join(' ') | replace('dist-git', 'scratch') }}"
+        build_package_tito_releasers: "{{ releasers | join(' ') | replace('dist-git', 'scratch') }}"
+      when: build_package_koji_command == 'brew'
       tags:
         - always
 
-    - include: tasks/tito_release.yml
+    - name: 'Set tito_releasers'
+      set_fact:
+        build_package_tito_releasers: "{{ releasers | join(' ') }}"
+        build_package_tito_args: "{{ build_package_tito_args }} --scratch"
+      when: build_package_koji_command == 'koji'
+      tags:
+        - always
```

Comment:
Based on IRC, this is moving into the role I believe?

Reply:
Yup, hence the WIP